charts/linkerd2/templates/policy-crd.yaml

```diff
@@ -0,0 +1,290 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: servers.policy.linkerd.io
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+  labels:
+    linkerd.io/control-plane-ns: {{.Values.namespace}}
+spec:
+  group: policy.linkerd.io
+  names:
+    kind: Server
+    plural: servers
+    singular: server
+    shortNames: [srv]
+  scope: Namespaced
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+
+      # additionalPrinterColumns:
+      #   - jsonPath: .spec.port
+      #     name: podSelector
+      #     type: string
+      #   - jsonPath: .spec.podSelector
+      #     name: port
+      #     type: string
+
+      schema:
+        openAPIV3Schema:
+          type: object
+          required: [spec]
+          properties:
+            spec:
+              type: object
+              required:
+                - podSelector
+                - port
+              properties:
+
+                podSelector:
+                  type: object
+                  description: >-
+                    Selects pods in the same namespace.
+                  properties:
```

Comment:
Should we have `oneof: required[matchLabels], required[matchExpression]` like we do below for serverauth?

Reply:
Yes, I think so.